packages/salesforcedx-vscode-apex/test/views/fakeFiles.ts

```diff
@@ -0,0 +1,69 @@
+const FILE_0 = `public class file0 {
```

Comment:
@jkalloor0701 - Do we really need to repeat these kinds of tests here? We already tested the parsing/language server on the Apex LSP side.